server/app.js

```diff
@@ -56,6 +59,13 @@ if (process.env.NODE_ENV === 'production') {
   });
 }
 
+// POST request to upload image
+app.post('/images', upload.single('image'), async (req, res) => {
+  const file = req.file;
+  const result = await uploadToS3(file);
```

Comment:
Sorry, I might have missed this the first time. In a future PR, please make sure you handle errors here (maybe use a try/catch) 😄